tests/integ/test_df_analytics.py

```diff
@@ -315,6 +335,10 @@ def custom_formatter(input_col, agg):
 
 
 @pytest.mark.skipif(not is_pandas_available, reason="pandas is required")
+@pytest.mark.skipif(
+    "config.getvalue('local_testing_mode')",
+    reason="FEAT: cast to float type not supported",
```

Comment:
We support to_double, so this is a bug, we should route reference to to_float to to_double.